internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -575,6 +575,12 @@ func (c *Coordinator) Upgrade(ctx context.Context, version string, sourceURI str
 	cb, err := c.upgradeMgr.Upgrade(ctx, version, sourceURI, action, det, skipVerifyOverride, skipDefaultPgp, pgpBytes...)
 	if err != nil {
 		c.ClearOverrideState()
+		if errors.Is(err, upgrade.ErrUpgradeSameVersion) {
+			c.logger.Info("Same ver upgrade detected.")
```

Comment:
dont't we end up with 2 log levels for same thing one Warning coming from `internal/pkg/agent/application/upgrade/upgrade.go:L175` the other one here